components/brave_vpn/browser/connection/mac/brave_vpn_os_connection_api_mac.mm

```diff
@@ -73,9 +73,12 @@ OSStatus RemoveKeychainItemForAccount() {
   return result;
 }
 
-OSStatus StorePassword(const NSString* password) {
+OSStatus StorePassword(const NSString* password, std::string* error_str) {
+  DCHECK(error_str);
+  std::string error;
   if (password == nil || [password length] == 0) {
-    LOG(ERROR) << "Error: password is empty";
+    *error_str = "Error: password is empty";
+    LOG(ERROR) << *error_str;
```

Comment:
do we actually need this log? can we replace multiple low level logs by one for last error when it set?